qutebrowser/mainwindow/prompt.py

```diff
@@ -642,8 +643,16 @@ def _init_fileview(self):
         self._file_model.directoryLoaded.connect(
             lambda: self._file_model.sort(0))
 
+    def _show_error(self, msg):
+        log.prompt.error(msg)
+        QToolTip.showText(self._lineedit.mapToGlobal(QPoint(0, 0)), msg)
+
```

Comment:
You can simply switch the order in which overlays are added in `mainwindow.py` to show the errors in front of the prompt - I think that'd be preferable.

If we want to keep this for some reason, it should be in `_BasePrompt` as it could be useful in the future for other prompts as well.